#!/usr/bin/env python
PACKAGE = "snailbot_driver"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("K_left_P", double_t, 0, "Left Motor Proportional Gain", 50, 0, 500)
gen.add("K_left_I", double_t, 0, "Left Motor Integral Gain", 25, 0, 100)
gen.add("K_left_D", double_t, 0, "Left Motor Derivative Gain", 10, 0, 100)
gen.add("K_right_P", double_t, 0, "Right Motor Proportional Gain", 50, 0, 500)
gen.add("K_right_I", double_t, 0, "Right Motor Integral Gain", 25, 0, 100)
gen.add("K_right_D", double_t, 0, "Right Motor Derivative Gain", 10, 0, 100)

exit(gen.generate(PACKAGE, "snailbot_driver", "MotorGains"))
